src/core/ollayerprimitive.js

```diff
@@ -10,7 +10,11 @@ goog.provide('olcs.core.OlLayerPrimitive');
  * @extends {Cesium.PrimitiveCollection}
  */
 olcs.core.OlLayerPrimitive = function(layerProjection, scene) {
-  goog.base(this);
+  if (!olcs.core.OlLayerPrimitive.initialized) {
+    olcs.core.OlLayerPrimitive.initialized = true;
+    goog.inherits(olcs.core.OlLayerPrimitive, Cesium.PrimitiveCollection);
+  }
+  Cesium.PrimitiveCollection.call(this);
```

Comment:
In this case `goog.base()` will just be replaced by the call to the super constructor.
This change is required due to the move of the call to `goog.inherits()` in order to workaround a verification of the closure compiler.